<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(31, 149, 239);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="resources.qrc">:/images/profile_pic.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>15</number>
         </property>
         <item>
          <widget class="QPushButton" name="dashbord_1">
           <property name="styleSheet">
            <string notr="true">QPushButton {
color:white;
height:30px;
border:none;
border-radius:10px;
}

QPushButton:checked {
background-color:#F5FAFE;
color:#1F95EF;
font-weight:bold;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/dashboard_white.png</normaloff>
             <normalon>:/images/dashboard.png</normalon>:/images/dashboard_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="message_1">
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color:white;
height:30px;
border:none;
border-radius:10px;
}

QPushButton:checked {
background-color:#F5FAFE;
color:#1F95EF;
font-weight:bold;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/profile_white.png</normaloff>
             <normalon>:/images/profile.png</normalon>:/images/profile_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="message_3">
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color:white;
height:30px;
border:none;
border-radius:10px;
}

QPushButton:checked {
background-color:#F5FAFE;
color:#1F95EF;
font-weight:bold;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/messages_white.png</normaloff>
             <normalon>:/images/messages.png</normalon>:/images/messages_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="notification_1">
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color:white;
height:30px;
border:none;
border-radius:10px;
}

QPushButton:checked {
background-color:#F5FAFE;
color:#1F95EF;
font-weight:bold;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/notifications_white.png</normaloff>
             <normalon>:/images/notifications.png</normalon>:/images/notifications_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="settings_1">
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color:white;
height:30px;
border:none;
border-radius:10px;
}

QPushButton:checked {
background-color:#F5FAFE;
color:#1F95EF;
font-weight:bold;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/settings_white.png</normaloff>
             <normalon>:/images/settings.png</normalon>:/images/settings_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>224</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_6">
         <property name="styleSheet">
          <string notr="true">QPushButton {
	color:white;
height:30px;
border:none;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/images/log_out_white.png</normaloff>:/images/log_out_white.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(31, 149, 239);

QPushButton {
color:white;
text-align:left;
height:30px;
border:none;
}

QPushButton:checked {
background-color:#F5FAFE;
color:#1F95EF;
font-weight:bold;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="rightMargin">
        <number>20</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="rightMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="resources.qrc">:/images/profile_pic.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Analyse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>15</number>
         </property>
         <item>
          <widget class="QPushButton" name="dashbord_2">
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color:white;
	text-align:left;
	height:30px;
border:none;
padding-left:10px;
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
}

QPushButton:checked {
background-color:#F5FAFE;
color:#1F95EF;
font-weight:bold;
}</string>
           </property>
           <property name="text">
            <string>Dashbord</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/dashboard_white.png</normaloff>
             <normalon>:/images/dashboard.png</normalon>:/images/dashboard_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="message_2">
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color:white;
	text-align:left;
	height:30px;
border:none;
padding-left:10px;
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
}

QPushButton:checked {
background-color:#F5FAFE;
color:#1F95EF;
font-weight:bold;
}</string>
           </property>
           <property name="text">
            <string>Profile</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/profile_white.png</normaloff>
             <normalon>:/images/profile.png</normalon>:/images/profile_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="message_4">
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color:white;
	text-align:left;
	height:30px;
border:none;
padding-left:10px;
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
}

QPushButton:checked {
background-color:#F5FAFE;
color:#1F95EF;
font-weight:bold;
}</string>
           </property>
           <property name="text">
            <string>Message</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/messages_white.png</normaloff>
             <normalon>:/images/messages.png</normalon>:/images/messages_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="notification_2">
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color:white;
	text-align:left;
	height:30px;
border:none;
padding-left:10px;
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
}

QPushButton:checked {
background-color:#F5FAFE;
color:#1F95EF;
font-weight:bold;
}</string>
           </property>
           <property name="text">
            <string>Notification</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/notifications_white.png</normaloff>
             <normalon>:/images/notifications.png</normalon>:/images/notifications_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="settings_2">
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color:white;
	text-align:left;
	height:30px;
border:none;
padding-left:10px;
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
}

QPushButton:checked {
background-color:#F5FAFE;
color:#1F95EF;
font-weight:bold;
}</string>
           </property>
           <property name="text">
            <string>Settings</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/settings_white.png</normaloff>
             <normalon>:/images/settings.png</normalon>:/images/settings_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>224</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_7">
         <property name="styleSheet">
          <string notr="true">QPushButton {
	color:white;
	text-align:left;
	height:30px;
border:none;
padding-left:10px
}</string>
         </property>
         <property name="text">
          <string>Sign Out</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/images/log_out_white.png</normaloff>:/images/log_out_white.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QWidget" name="widget_3" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(245, 240, 254);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QWidget" name="header_widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="pushButton_13">
            <property name="styleSheet">
             <string notr="true">QPushButton {
	color:white;
	border:none;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/images/menu.png</normaloff>:/images/menu.png</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_14">
              <property name="styleSheet">
               <string notr="true">QPushButton {
	color:white;
	border:none;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/images/search.png</normaloff>:/images/search.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_15">
            <property name="styleSheet">
             <string notr="true">QPushButton {
	color:white;
	border:none;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/images/image.png</normaloff>:/images/image.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <widget class="QWidget" name="dashbord_page">
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>320</x>
             <y>210</y>
             <width>231</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Dashbord page</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="profile_page">
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>330</x>
             <y>220</y>
             <width>231</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Profile page</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="message_page">
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>340</x>
             <y>220</y>
             <width>231</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Message page</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="notification_page">
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>320</x>
             <y>200</y>
             <width>231</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Notification page</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="settings_page">
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>320</x>
             <y>210</y>
             <width>231</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Settings page</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_13</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>51</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_13</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>settings_1</sender>
   <signal>toggled(bool)</signal>
   <receiver>settings_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>notification_1</sender>
   <signal>toggled(bool)</signal>
   <receiver>notification_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>message_3</sender>
   <signal>toggled(bool)</signal>
   <receiver>message_4</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>message_1</sender>
   <signal>toggled(bool)</signal>
   <receiver>message_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dashbord_1</sender>
   <signal>toggled(bool)</signal>
   <receiver>dashbord_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>90</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dashbord_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>dashbord_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>message_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>message_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>43</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>message_4</sender>
   <signal>toggled(bool)</signal>
   <receiver>message_3</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>notification_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>notification_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>29</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>settings_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>settings_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>36</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>19</x>
     <y>557</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>659</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>552</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>623</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
